examples/ovs/README.md

```diff
@@ -11,6 +11,26 @@ Contain basic setup for NSM that includes `nsmgr`, `forwarder-ovs`, `registry-k8
 - [Kernel to Kernel Connection](../use-cases/Kernel2Kernel)
 - [SmartVF to SmartVF Connection](../use-cases/SmartVF2SmartVF)
 
+## SR-IOV config
```

Comment:
@Bolodya1997 The SR-IOV config is bit different for SmartNIC case, Can you just follow this [documentation](https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/docs/ovs-offload.md#mellanox-sr-iov-configuation) (except OVS bridge configuration) ?

Reply:
This PR is about **what should be SR-IOV config file** not about **how NICs should be configured**, so I guess such additional information can be added with some another PR if you want :)

Reply:
@Bolodya1997 Sure, I get it.